v2/sourcedb-to-spanner/README_Sourcedb_to_Spanner_Flex.md

```diff
@@ -125,65 +113,63 @@ need valid resources for the required parameters.
-export DEFAULT_LOG_LEVEL=INFO
+# Adding timestamp just allows the name to be unique across multiple runs
+JOB_NAME="<job-name>-`date +'%Y-%m-%d-%H-%M-%S-%N'`"
+TEMPLATE_NAME="Sourcedb_to_Spanner_Flex"
+PROJECT="<GCP-PROJECT-NAME>"
+
+# Note this changes the default project of your bash session.
+gcloud config set project $PROJECT
+
+
+# Below are GCS paths, they could be in same bucket separated by folders.
+TEMPLATE_BUCKET_NAME="<GCS_BUCKET_WHERE_TEMPLATE_IS_BUILT>"
+TEMPLATE_PATH="${TEMPLATE_BUCKET_NAME}/templates/flex/${TEMPLATE_NAME}"
+DLQ_DIRECTORY="<GCS_PATH_FOR_DLQ>"
+
+
+# Source Parameters
+SRC_USER_ID="<SOURCE_DB_USER_ID>"
+SRC_PWD="<SOURCE_PASSWORD>"
+SRC_IP="<SOURCE_ID>"
+SOURCE_HOST="${SRC_IP}"
+SOURCE_PORT="3306"
+SOURCE_DB="<NAME_OF_SOURCE_DB>"
+
+# Spanner details
+SPANNER_INSTANCE_ID="<SPANNER_INSTANCE_ID>"
+SPANNER_DB_ID="<SPANNER_DB_ID>"
+
+
+# Dataflow Parameters
+WORKER_REGION="<REGION_TO_SPAWN_DATAFLOW_WORKERS like us-central1>"
+NETWORK="<VPC network to spawn dataflow workers>"
+SUBNETWORK="regions/${WORKER_REGION}/subnetworks/<SUBNET_NAME>"
+
+# Typical Configuration for dataflow scaling, please change as per load
+MAX_WORKERS=10
+NUM_WORKERS=10
+WORKER_MACHINE_TYPE="n1-highmem-32"
+# Override NUM_PARTITIONS for tall tables.
+# NUM_PARTITIONS="4000"
 
 gcloud dataflow flex-template run "sourcedb-to-spanner-flex-job" \
   --project "$PROJECT" \
   --region "$REGION" \
   --template-file-gcs-location "$TEMPLATE_SPEC_GCSPATH" \
-  --parameters "jdbcDriverJars=$JDBC_DRIVER_JARS" \
-  --parameters "jdbcDriverClassName=$JDBC_DRIVER_CLASS_NAME" \
-  --parameters "sourceHost=$SOURCE_HOST" \
-  --parameters "sourcePort=$SOURCE_PORT" \
-  --parameters "sourceDB=$SOURCE_DB" \
-  --parameters "sourceConnectionProperties=$SOURCE_CONNECTION_PROPERTIES" \
+  --parameters "sourceHost=jdbc:mysql://${SOURCE_IP}:${SOURCE_PORT}/${SOURCE_DB}" \
   --parameters "username=$USERNAME" \
   --parameters "password=$PASSWORD" \
-  --parameters "tables=$TABLES" \
-  --parameters "numPartitions=$NUM_PARTITIONS" \
-  --parameters "fetchSize=$FETCH_SIZE" \
   --parameters "instanceId=$INSTANCE_ID" \
   --parameters "databaseId=$DATABASE_ID" \
   --parameters "projectId=$PROJECT_ID" \
+  --parameters "jdbcDriverClassName=$JDBC_DRIVER_CLASS_NAME" \
+  --parameters "tables=$TABLES" \
+  --parameters "numPartitions=$NUM_PARTITIONS" \
   --parameters "spannerHost=$SPANNER_HOST" \
-  --parameters "ignoreColumns=$IGNORE_COLUMNS" \
   --parameters "maxConnections=$MAX_CONNECTIONS" \
-  --parameters "reconnectsEnabled=$RECONNECTS_ENABLED" \
-  --parameters "reconnectAttempts=$RECONNECT_ATTEMPTS" \
   --parameters "sessionFilePath=$SESSION_FILE_PATH" \
-  --parameters "disabledAlgorithms=$DISABLED_ALGORITHMS" \
-  --parameters "extraFilesToStage=$EXTRA_FILES_TO_STAGE" \
-  --parameters "defaultLogLevel=$DEFAULT_LOG_LEVEL"
+  --parameters "DLQDirectory=$DLQ_DIRECTORY"
 ```
 
 For more information about the command, please check:
```

Comment:
On similar lines of above, are these autogenerated, or is the source a standard way of classifying the parameters across tempaltes?

Reply:
this file has been auto generated now.